blaze/server/server.py

```diff
@@ -562,10 +600,13 @@ def log_time(start=time()):
                                         expr.dshape,
                                         odo_kwargs)
         except NotImplementedError as e:
-            return ("Computation not supported:\n%s" % e, RC.NOT_IMPLEMENTED)
+            error_msg = "Computation not supported:\n%s\n%s" % (e, traceback.format_exc())
+            app.logger.error(error_msg)
+            return (error_msg, RC.NOT_IMPLEMENTED)
         except Exception as e:
-            return ("Computation failed with message:\n%s: %s" % (type(e).__name__, e),
-                    RC.INTERNAL_SERVER_ERROR)
+            error_msg = "Computation failed with message:\n%s: %s\n%s" % (type(e).__name__, e, traceback.format_exc())
```

Comment:
Could we make the `format_exc` function configurable? The use case that we have is that we run an extension of blaze server which may potentially have sensitive information in the traceback, we would like to filter the information the client sees before sending it over. I was thinking a function from traceback to str.

To pass the traceback to the function in a py2/3 compat way we could do:

```python
    tbmsg = serialize_traceback(sys.exc_info()[2])  # it is important not to store exc_info or the tb in a local
```

By default it could be `toolz.compose(''.join, traceback.format_tb)` which will do the same as it is currently doing.

Also, how do you feel about printing the traceback and then the  error. Right now the format will look like:

```
client tb
server  error message
server tb
```

and I think it may be nicer as:

```
client tb
server tb
server error message
```